packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/EditLibraryView.types.ts

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { GQLLibrary } from './useLibraries.types';
+
+export type EditLibraryViewParams = 'namespace' | 'name' | 'version';
+
+export type EditLibraryViewState = {
+  data: GQLGetLibraryQueryData;
+};
+
+export interface GQLGetLibraryQueryData {
+  viewer: GQLViewer;
+}
+
+export interface GQLViewer {
+  library: GQLLibrary | null;
+}
+
+export interface GQLGetLibrariesQueryVariables {
+  name: string;
+  namespace: string;
+  version: string;
+}
```

Comment:
This has nothing to do here, just reuse the existing type because this is what you will have

Reply:
Done